### ISSUE TITLE
Clear selection model when press Go button on TagLink

### ISSUE THREAD

**Issue by knalum** [knalum/YATA#2]
When pressing Go button on TagLink, clear the selection model before selecting the source Tag.

**Comment by knalum** [comment]
![screen shot 2016-04-25 at 9 33 49 pm](https://cloud.githubusercontent.com/assets/7584766/14796405/6c98c844-0b2d-11e6-9e49-442efef79ee0.png)

**Comment by knalum** [comment]
Solved in 6e964aa 
```java
        this.goToSourceTag.setOnAction(new EventHandler<ActionEvent>() {
            @Override
            public void handle(ActionEvent event) {
                TagGroup node = findTagInTree(getSourcePath());
                getModelTree().getSelectionModel().clearSelection();
                getModelTree().getSelectionModel().select(node);
            }
        });
```
